main.py

```diff
@@ -0,0 +1,11 @@
+import random 
+
+if __name__ == "__main__":
+   while True:     
+      print(''' 1. roll the dice             2. exit     ''')    
```

Comment:
Having the user types y/n for input could be less confusing than typing a number and seeing another number as the response. 
Consider the example:
$ python main.py
 1. roll the dice             2. exit     
what you want to do
1
2
 1. roll the dice             2. exit
what you want to do
1
2
 1. roll the dice             2. exit
what you want to do
2